pkg/doc/verifiable/example_credential.go

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
This should be a `_test` file (ie. not in production code)

Reply:
OK!